systems-ui/src/pages/system-detail/SystemDetail.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import SystemsOrchestrator from "../../common/components/systems/systems-orchestrator/SystemsOrchestrator";
+
+class SystemDetail extends Component {
+  render() {
+    return (
+      <SystemsOrchestrator>
+        {
+          ({getProductById}) => {
+            const systemDetail = getProductById(1000010000)
+
+            return (
+              <div className='container-fluid'>
+                <h1 className="display-1">System {systemDetail.id}</h1>
+                <dl className="row">
+                  <dt className="col-sm-3">biosversion</dt>
```

Comment:
This works fine but could be a bit static.
How would you approach a refactor to something more dynamic in case the system detail set is unpredictably different for each system?

Reply:
I am not sure I understood the question here. Probably I would try to group it in logic sections to compact it more and prioritize the most important section to the top.